platform/ios/MapLibre.docc/MapLibre.md

```diff
@@ -18,6 +18,7 @@ Powerful, free and open-source mapping toolkit with full control over data sourc
 - <doc:LocationPrivacyExample>
 - <doc:BlockingGesturesExample>
 - <doc:LineStyleLayerExampleUIKit>
+- <doc:WebAPIDataExampleUIKit>
```

Comment:
Can this part be auto generated somehow? I see that you add a section here every time you add a new page...

Reply:
No, but I will manually sort the examples by category when I am done.